datafeeder/src/main/resources/md_record_template.xml

```diff
@@ -190,11 +190,8 @@
       <gmd:language>
         <gmd:LanguageCode codeList="http://www.loc.gov/standards/iso639-2/" codeListValue="eng" />
       </gmd:language>
-      <!-- 
       <gmd:topicCategory>
```

Comment:
What is this file used to ? Because the template comes from the datadir doesn't it ?